tests/functional/aws-node-sdk/test/object/websiteGet.js

```diff
@@ -504,20 +419,17 @@ describe('User visits bucket website endpoint', () => {
                 };
                 const redirect = {
                     HostName: 'www.google.com',
-                    ReplaceKeyPrefixWith: '/about',
+                    ReplaceKeyPrefixWith: 'about/',
```

Comment:
yes -- think we had a typo before as it was redirecting to `http://google.com//about` (and it was passing the test because google then redirects to `https://www.google.com/about/`)